TankLib/STU/Types/Misc.cs

```diff
@@ -3224,8 +3224,10 @@ public class STU_0FA0034C : STUInstance
         public uint m_wwiseEventIDStop;
         
         [STUField(0xF7805734, 164)] // size: 4
-        public uint m_F7805734;
-        
+        //Sound class.
+        //3851462253 is loading screen music
+        public uint m_soundClass;
+
```

Comment:
this file cannot be edited, it is automatically generated, any changes made here will get undone when the file is regenerated causing errors in code that references it.

Reply:
also it's probably sound trigger, not sound class.

Reply:
I think that m_wwiseEventIDPlay is a trigger :)

Reply:
The name of the field is `m_wwiseBankID`, all the files with the same id also have the same STUSoundBank GUID. You can change it to that.

Reply:
gotcha